JPS_Ontology/ontology/ontodevice/OntoDevice.csv

```diff
@@ -50,14 +52,26 @@ hasSystemCapability,Object Property,,,Device,http://www.w3.org/ns/ssn/systems/Sy
+hasPressure,Object Property,hasQuantity,IS-A,https://w3id.org/bot#Zone,http://www.ontology-of-units-of-measure.org/resource/om-2/Pressure,,"A relationship describing the pressure exerted on a substance, object or a space.",https://www.theworldavatar.com/kg/ontodevice/,has pressure
+hasPressure,Object Property,hasQuantity,IS-A,http://www.theworldavatar.com/ontology/ontocape/material/phase_system/phase_system.owl#SinglePhase,http://www.ontology-of-units-of-measure.org/resource/om-2/Pressure,,"A relationship describing the pressure exerted on a substance, object or a space.",https://www.theworldavatar.com/kg/ontodevice/,has pressure
 isAttachedTo,Object Property,,,Device,https://w3id.org/bot#Element,,A relationship that describes a device being attached to another physical element or object.,https://www.theworldavatar.com/kg/ontodevice/,is attached to
 isConnectedTo,Object Property,,,Device,Device,,A relationship that describes a device being connected to another device where the operation of one device will affect the other.,https://www.theworldavatar.com/kg/ontodevice/,is connected to
 sendsSignalTo,Object Property,,,Device,Device,,A relationship that describes a device sending a signal to another device.,https://www.theworldavatar.com/kg/ontodevice/,sends signal to
 hasGeoLocation,Object Property,,,Device,http://www.opengis.net/ont/sf#Point,,A relationship that describes a device's location.,https://www.theworldavatar.com/kg/ontodevice/,has geo location
-hasProperty,Object Property,,,Device,Property,,A relationship that describes a property of a device or space.,https://www.theworldavatar.com/kg/ontodevice/,has property
-hasProperty,Object Property,,,https://w3id.org/bot#Zone,Property,,A relationship that describes a property of a device or space.,https://www.theworldavatar.com/kg/ontodevice/,has property
+sensesSubstance,Object Property,,,FlowSensor,http://www.theworldavatar.com/ontology/ontocape/material/substance/substance.owl#Substance,,A relationship that describes the type of substance that a device is sensing or observing.,https://www.theworldavatar.com/kg/ontodevice/,senses substance
+sensesSubstance,Object Property,,,PressureSensor,http://www.theworldavatar.com/ontology/ontocape/material/substance/substance.owl#Substance,,A relationship that describes the type of substance that a device is sensing or observing.,https://www.theworldavatar.com/kg/ontodevice/,senses substance
+sensesSubstance,Object Property,,,https://saref.etsi.org/core/TemperatureSensor,http://www.theworldavatar.com/ontology/ontocape/material/substance/substance.owl#Substance,,A relationship that describes the type of substance that a device is sensing or observing.,https://www.theworldavatar.com/kg/ontodevice/,senses substance
```

Comment:
For several properties, e.g. sensesSubstance, hasTemperature, hasPressure, hasQuantity, you're each making multiple independent definitions with different domains/ranges. Please note that the domain/range of each of these properties will then be interpreted as _intersection_ of all the domains/ranges (which is certainly incorrect). You will need to replace these with single definitions using the UNION keyword for the domains/ranges, possibly followed by definitions that restrict the properties on the subclasses of the domains using the `only` quantifier.